modules/core/src/lib/layer.js

```diff
@@ -680,47 +680,23 @@ export default class Layer extends Component {
   // Dirty some change flags, will be handled by updateLayer
   /* eslint-disable complexity */
   setChangeFlags(flags) {
-    this.internalState.changeFlags = this.internalState.changeFlags || {};
-    const changeFlags = this.internalState.changeFlags;
+    const {changeFlags} = this.internalState;
 
-    // Update primary flags
-    if (flags.dataChanged && !changeFlags.dataChanged) {
-      changeFlags.dataChanged = flags.dataChanged;
-      debug(TRACE_CHANGE_FLAG, this, 'dataChanged', flags);
-    }
-    if (flags.updateTriggersChanged && !changeFlags.updateTriggersChanged) {
-      changeFlags.updateTriggersChanged =
-        changeFlags.updateTriggersChanged && flags.updateTriggersChanged
-          ? Object.assign({}, flags.updateTriggersChanged, changeFlags.updateTriggersChanged)
```

Comment:
No breaking change here - this merge never happens due to the if statement above